licenses/library.go

```diff
@@ -80,9 +80,13 @@ func Libraries(ctx context.Context, classifier Classifier, importPaths ...string
 	pkgsByLicense := make(map[string][]*packages.Package)
 	errorOccurred := false
 	packages.Visit(rootPkgs, func(p *packages.Package) bool {
-		if len(p.Errors) > 0 {
-			errorOccurred = true
-			return false
+		for _, err := range p.Errors {
+			// Ignore errors complaining that the package is a binary. We still want
+			// to traverse related packages.
+			if !strings.Contains(err.Msg, "is a program, not an importable package") {
```

Comment:
I think this should be controlled by a flag, since I don't think all users will want this behaviour. For my use cases, for example, I only want to include dependencies linked into a binary, not the dependencies of the tooling. What is the your use case that requires this?

Reply:
For our project, we originally we though we were going to need to include things like code generators. After doing some digging, turns out this isn't necessary.

Went ahead an added the flag anyway, since this feels like it might be a useful flag even though I don't have a direct use case for it at the moment. Let me know what you think / if you want to move forward with this. We can always sit and add it later if it comes up.